### ISSUE TITLE
Seeing many blank aria-labelledby attributes in BASE

### ISSUE THREAD

**Issue by terracoda** [phetsims/a11y-research#108]
@jessegreenberg, @zepumph, @mbarlow12, I am seeing blank `aria-labelledby=""` attributes on all elements in BASE. I'm not sure why these attributes have appeared en masse.

Are you guys working on something? 
Is something in transition?
Perhaps, there is an open issue in another repo about this?

**Comment by terracoda** [comment]
**Screen shot from BASE on Master**
<img width="675" alt="screen shot 2018-06-11 at 10 30 08 am" src="https://user-images.githubusercontent.com/1902116/41233342-bea59aa0-6d63-11e8-87f5-adc74b4f8a5a.png">

**Comment by terracoda** [comment]
Has anyone checked to see if null strings are allowed (valid code) for `aria-labelledby`?

**Comment by jessegreenberg** [comment]
Hmm probably caused by changes in https://github.com/phetsims/scenery/issues/701

**Comment by jessegreenberg** [comment]
Thanks for catching this @terracoda, should be fixed in the above commit, those blank aria-labelledby attributes have been removed.

**Comment by terracoda** [comment]
I am totally not up to speed on what you guys are implementing, but I am wondering if you considered options for creating an accessible name? Perhaps this is too high-level. 
- option 1: inner text of element, e.g., 
```html
<button>My Button's Name</button>
```
- option 2: using `for` attribute on a `label` element to associate an html `label` with html control element, e.g., 
``` html
<input id="radiobuttonIDREF" type="radio"><label for="radiobuttonIDREF">My Radio Button's Name</label>
```
- option 3: using `aria-label` attribute to label an html element that does not have inner text, or label or other way to provide an accessible name, e.g., 
``` html
<input id="checkboxIDREF" type="checkbox" aria-label="My Checkbox's Name">
```
- option 4: using `aria-labelledby` attribute to label an html element that can be associated with an element in the PDOM, like a heading element that would provide the elements name inner text, e.g., 
``` html
<section id="sectionIDREF" aria-labelledby="headingIDREF">
<h2 id="headingIDREF">My Section's Name</h2>
....
</section> 
```
Of course, examples are simplified, and I am not trying to imply an order of priority with these options. There may be diverse reasons (interaction preferences, pedagogical reasons, technical reasons, visual design) on which naming option to choose.

**Comment by terracoda** [comment]
Thanks @jessegreenberg, I'll close again as the issue has been resolved, but leave my comment.
